.github/actions/commit-status/end/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: CommitStatusEnd
```

Comment:
til that commit statuses existed: https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28